src/components/Departments/DepartmentDetailComponent.jsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import { isEmpty } from 'lodash';
+import PropTypes from 'prop-types';
+import { Header, Card, Divider, Segment, Grid } from 'semantic-ui-react';
+import { Link } from 'react-router-dom';
+import LoaderComponent from '../../components/LoaderComponent';
+import NavBarComponent from '../../_components/NavBarContainer';
+import '../../_css/UserDetailComponent.css';
+
+
+export class DepartmentDetailComponent extends React.Component {
+  componentDidMount() {
+    const { match, loadDepartmentDetail, departmentDetail } = this.props;
+    if (isEmpty(departmentDetail)) loadDepartmentDetail(+match.params.id);
+  }
+
+  viewAssignedAssets = assets =>
```

Comment:
`getAssetsSuccess` says nothing about what the method is doing. I'm struggling to understand it